### ISSUE TITLE
Watson document conversion server-side crashes when deals with html

### ISSUE THREAD

**Issue by ningyan325** [watson-developer-cloud/node-red-node-watson#161]
Please reference on disscussion at stackoverflow :  http://stackoverflow.com/questions/38293629/while-using-document-conversion-with-html-in-node-red-getting-error-lost-conne/38355094#comment64213696_38355094

**Comment by chughts** [comment]
Thanks for raising this. I will take a look. First I will need to reproduce.

**Comment by chughts** [comment]
Recreated - 

```
18 Jul 15:14:49 - [red] Uncaught Exception:
18 Jul 15:14:49 - TypeError: Cannot read property 'ext' of null
    at /Users/soheelchughtai/Documents/dev/node-red/wdc/node-red-node-   watson/services/document_conversion/v1.js:76:23
at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:380:3)
```

The input is identified as a url, but file-type is not recognizing the buffer. Fix will be to add a check and default the extension when file-type can't find one.

**Comment by chughts** [comment]
Fixed in #164
